internal/builders/docker/pkg/builder_test.go

```diff
@@ -53,7 +53,7 @@ func Test_CreateBuildDefinition(t *testing.T) {
 	}
 
 	if diff := cmp.Diff(got, want); diff != "" {
-		t.Errorf(diff)
+		t.Error(diff)
```

Comment:
lint: change `t.Errorf` to `t.Error` when not actually using any string formatting.